README.md

```diff
@@ -2,20 +2,33 @@
 
-While `window.print()` is a basic function that triggers the browser's default print dialog, the `navigator.printing` API provides a more comprehensive set of methods and options for querying local printers, submitting print jobs with customization options, managing print jobs, and customizing print workflows. This extended functionality allows developers to have more control over the printing process and enables advanced use cases, such as printing specific pages or sections of a document, setting print options like paper size and orientation, and monitoring the status of print jobs.
+### Streamlining Workflows
+First and foremost, this shift calls for a more streamlined and intuitive printing experience. Developers can design print workflows that automate repetitive tasks, preconfigure print settings with respect to a particular printer's capabilities, and eliminate unnecessary user interactions.
 
-### Leveraging Progressive Web Apps
-The new Web API can motivate developers to switch to Progressive Web Apps (PWAs) by offering enhanced printing capabilities that were traditionally limited to native applications (in particular, support for Printer Redirection).
+### Knowing Your Printers
+To enable customization of settings, developers need to have knowledge about the capabilities of the printer. The API provides a convenient method for accessing this information, which enhances the printing experience by allowing developers to retrieve relevant details about what the printer can and cannot do. A few examples:
+* If the printer supports high-resolution printing, the application can automatically select the appropriate print resolution, resulting in sharper and more detailed output.
+* If the printer supports duplex printing, the application can prompt the user to print double-sided, saving paper and reducing printing time.
+* Access to supported media sizes empowers the application to generate print-ready files by adjusting page layouts, margins, or scaling, effectively ensuring that the content fits properly on the printed page without any cropping or distortion.
+
+### Observing the Printing Pipeline
+When users utilize `window.print()`, they have limited visibility and control over the printing process - usually it's scoped to hidden system UI surfaces & system notifications that cannot be accessed by the app. The API strives to fill this gap by offering the ability to track the progress of print jobs as well as cancel them if needed directly to the applications, creating a more evident and clear printing experience via custom information surfaces.
```

Comment:
To back this up I would want to see an example of a native application which provides a significantly better user experience than what is provided by default by the platform. I would place this lower in the document as it is less compelling than the next two benefits.